arionBackend/api/__init__.py

```diff
@@ -1,3 +1,11 @@
 """
 This module contains all the APIs of the project.
 """
+from rest_framework.views import APIView
+
+
+class GetXByIdView(APIView):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a docstring to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1720)